GetSSL.ps1

```diff
@@ -36,6 +36,17 @@
 
 [cmdletbinding(SupportsShouldProcess=$true,ConfirmImpact='High',DefaultParameterSetName='InstallDependencies')]
 Param(
+	<#
+		Specify an alternate web root directory for HTTP-01 challenge. For use if FileMaker's web server
+		is not exposed to the internet, but another web server is. As long as that other server's
+		filesystem is accessible to the FileMaker server, anyway.
+
+		For example, -WebRoot '\\web-server\C$\inetpub\wwwroot'
+	#>
+	[Parameter(ParameterSetName='Setup')]
+	[Parameter(ParameterSetName='Renew')]

```

Comment:
I think it should be included in `ScheduleTask` and `InstallCertificate` parameter set's as well, shouldn't it?

Reply:
Oops, you're right about ScheduleTask. I pushed another commit to include that. I'm not sure about InstallCertificate, though. The function doesn't appear to do any challenging.